modules/navier_stokes/test/src/materials/AriaLaserWeld304LStainlessSteelBoundary.C

```diff
@@ -66,6 +66,8 @@ AriaLaserWeld304LStainlessSteelBoundary::AriaLaserWeld304LStainlessSteelBoundary
     _grad_surface_tension(declareADProperty<RealVectorValue>(
         getParam<MaterialPropertyName>("grad_surface_tension_name"))),
     _ad_normals(_assembly.adNormals()),
+    _ad_curvatures(_assembly.adCurvatures()),
```

Comment:
Have you done testing of this in 2D and 3D? I have some vague recollection that the curvature sign convention is different between 2D and 3D

Reply:
Kind of a gotcha which, if it exists, is embarrassing that I haven't documented or fixed

Reply:
I haven't tested this with 3D. 
Will do it in a few hours.

Reply:
To be honest, none of the tests use the curvature term at the moment. For that I'd need a second-order mesh. Will add a test for it in a second.

Reply:
Well, the libmesh source says this:
```
                 // The curvature is computed via the familiar Frenet formula:
                 // curvature = [d^2(x) / d (xi)^2] dot [normal]
                 // For a reference, see:
                 // F.S. Merritt, Mathematics Manual, 1962, McGraw-Hill, p. 310
                 //
                 // Note: The sign convention here is different from the
                 // 3D case.  Concave-upward curves (smiles) have a positive
                 // curvature.  Concave-downward curves (frowns) have a
                 // negative curvature.  Be sure to take that into account!
```